### DISCUSSION TITLE
Other Quarto thesis templates

### DISCUSSION THREAD

**Discussion by eeholmes** [nmfs-opensci/quarto-thesis#11] [General]
Here is a list of those I have seen

* https://github.com/numbats/monash-quarto-thesis
* https://github.com/PublicHealthDataGeek/UoL_quarto_thesis

Here is a list of some other Quarto PDF extensions that might be useful too

https://github.com/nmfs-opensci/quarto_titlepages/discussions/36

**Comment by PublicHealthDataGeek** [comment]
Hi Eli
I have developed a quarto thesis template based on yours that meets the University of Leeds (UK) thesis requirements.  It is available here:

[https://github.com/PublicHealthDataGeek/UoL_quarto_thesis](https://github.com/PublicHealthDataGeek/UoL_quarto_thesis)

Many thanks for such a great template and providing help and support whilst I was adapting for my university requirements.

Best wishes

Caroline

**Reply by eeholmes** [reply]
Nice job! I am glad my template was useful.

**Comment by nicole-brewer** [comment]
Hi there! I've made a list of all the Quarto extensions for university dissertations and theses that I could find on GitHub and put them in [Jupyter4Science/awesome-quarto-thesis](https://github.com/Jupyter4Science/awesome-quarto-thesis). I also included the template associated with this repository in a section about "creating your own template". I'm really excited to see some many people pursuing reproducible dissertations with Quarto! Thanks for putting this together @eeholmes

**Reply by nicole-brewer** [reply]
The extension for my university ([juliantao/quarto-asu-thesis](https://github.com/juliantao/quarto-asu-thesis)) also credits you as the inspiration for their template. Great stuff
